src/main/webapp/scripts/auth.js

```diff
@@ -0,0 +1,160 @@
+/**
+ * Copyright 2018 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+  // Your web app's Firebase configuration
+var firebaseConfig = {
+    apiKey: "AIzaSyCoNJFqs76_GbI1i1T7hhPmyxqv1Oc2hU4",
+    authDomain: "quizzy-step-2020.firebaseapp.com",
+    databaseURL: "https://quizzy-step-2020.firebaseio.com",
+    projectId: "quizzy-step-2020",
+    storageBucket: "quizzy-step-2020.appspot.com",
+    messagingSenderId: "1029940211712",
+    appId: "1:1029940211712:web:8e31fe1fcb8423d4a728e5",
+    measurementId: "G-M62T74PK0V"
+};
+  
+// Initialize Firebase
+firebase.initializeApp(firebaseConfig);
```

Comment:
I think its really nice that you extracted the Auth thing to its own file! I have a question though, if we wanted to do more stuff with Firebase like querying things, will we have to do all of that in this file or what happens if we initialize Firebase for the same page in two diferent JS files? Like if we had a `queries.js`and this `auth.js`, do you think there would be a problem?

Reply:
We can just run the `auth.js` script before any other script so we can run `queries.js` so it signs in before doing anything.